src/dodal/devices/slits.py

```diff
@@ -2,16 +2,65 @@
 from ophyd_async.epics.motor import Motor
 
 
-class Slits(StandardReadable):
+class MinimalSlits(StandardReadable):
+    """Gap only X Y slits."""
+
+    def __init__(
+        self,
+        prefix: str,
+        suffix: tuple[str, str] = ("X:SIZE", "Y:SIZE"),
+        name: str = "",
+    ) -> None:
+        with self.add_children_as_readables():
+            self.x_gap = Motor(prefix + suffix[0])
+            self.y_gap = Motor(prefix + suffix[1])
```

Comment:
```suggestion
        x_gap: str = "X:SIZE",
        y_gap: str = "Y:SIZE",
        name: str = "",
    ) -> None:
        with self.add_children_as_readables():
            self.x_gap = Motor(prefix + x_gap)
            self.y_gap = Motor(prefix + y_gap)
```

I think this became unwieldy fast, and without information on what each is used for. This way also allows overriding particular values without overriding all of them.